### PR TITLE
build-ha-io: fix left/right interface parameters names

### PR DESCRIPTION
**Pull request by andriytk**
The parameter names should be with a hyphen (not underscore),
similar to all the rest parameters names and aligned with the doc.

### CODE REVIEW

File: conf/script/build-ha-io [hunk 1]
```diff
@@ -144,8 +144,8 @@ if [[ -f $argsfile ]]; then
            ip1)          ip1=$value     ;;
            ip2)          ip2=$value     ;;
            interface)    iface=$value   ;;
-           left_iface)   left_iface=$value   ;;
-           right_iface)  right_iface=$value   ;;
+           left-iface)   left_iface=$value   ;;
+           right-iface)  right_iface=$value   ;;
```
Comment (indrajitzagade) [inline conf/script/build-ha-io:148]: Do we need to change at other places too in this file? @andriytk

Comment (andriytk) [reply conf/script/build-ha-io:148]: Like where?

Comment (indrajitzagade) [reply conf/script/build-ha-io:148]: sorry, never mind

**Review by indrajitzagade — APPROVED** [review]

**Comment by indrajitzagade** [comment]
@mssawant @madhavemuri Please review.

File: conf/script/build-ha-io (hunk 1 above)
Comment (ypise) [reply conf/script/build-ha-io:148]: @andriytk 
We need change on line: https://github.com/Seagate/cortx-ha/pull/159/files#diff-7801cbb90727f0c5f7c79eac586ff9c6R96

Comment (ypise) [reply conf/script/build-ha-io:148]: That's taken care of, just checked the code.
We are good.

**Review by ypise — APPROVED** [review]

**Comment by andriytk** [comment]
@indrajitzagade , two approvals are there - please land this.

**Review by saurabhkhanvilkar — APPROVED** [review]

**Pull request merged**